devfile.yaml

```diff
@@ -9,8 +9,6 @@
 schemaVersion: 2.2.2
 metadata:
   name: che-docs
-attributes:
-  controller.devfile.io/storage-type: ephemeral
```

Comment:
why would you like to remove it?

Reply:
I am using Che for daily work and was surprised all my changes were gone after workspace restart.
Do you want me to remove this change from this PR and discuss it later?

Reply:
it's fine to remove the ephemeral attribute 👍 I normally start in ephemeral and do commit/push